README.md

```diff
@@ -47,9 +47,14 @@ variations.
 
 Each test case has the the following keys:
 - description: which will be used to name each generated test
+  - the description should not simply explain **what** each case is (that is redundant information)
+  - the description should explain **why** each case is there
+  - for example, what kinds of incorrect implementations might this case help us find?
 - 'variable names': one or more variable names with values which will be passed to the solution method
```

Comment:
I was wondering about the "will be passed to the solution method" part. Shouldn't that be "will be passed to the test method"?